src/dbnode/persist/fs/seek_manager.go

```diff
@@ -218,11 +232,19 @@ func (m *seekerManager) CacheShardIndices(shards []uint32) error {
 		byTime.accessed = true
 		byTime.Unlock()
 
-		if err := m.openAnyUnopenSeekersFn(byTime); err != nil {
-			multiErr = multiErr.Add(err)
-		}
+		wg.Add(1)
```

Comment:
More of a general question, but is it better to add 1 to `wg` every time or `wg.Add(len(shards))` up front?

Reply:
II tend to avoid doing `wg.Add(x)` upfront since sometimes there is a conditional `if foo { continue; }` which means the number is sometimes off.  And since the amount of work that happens asynchronously is very expensive, doesn't hurt on the lock acquisition side too much at all.